CMakeLists.txt

```diff
@@ -74,9 +80,9 @@ endif()
 
 find_package(GSL)
 find_package(LAPACK)
-find_package(Casacore)
+find_package(Casacore REQUIRED)
 find_package(Boost)
-find_package(Eigen3 REQUIRED)
+find_package(Eigen3 REQUIRED HINTS /usr/local/include/)
```

Comment:
I'd assume `/usr/local` is already considered for package searching, so it seems unnecessary. Also `/usr/local/include` should contain only C/C++ header files, so no cmake package should be found there.